src/components/Theme/tokens.scss

```diff
@@ -137,19 +136,13 @@
   );
 
   /* Back, Close (modal) icon button colors */
-  --osdk-color-background-button-icon-hover: rgba(var(--ods-color-neutral-400));
-  --osdk-color-background-button-icon-active: rgba(
-    var(--ods-color-neutral-600)
-  );
+  --osdk-color-background-button-icon-hover: rgb(var(--ods-color-neutral-400));
+  --osdk-color-background-button-icon-active: rgb(var(--ods-color-neutral-600));
 
   /* Selfie Camera button colors */
-  --color-background-button-camera: rgba(var(--ods-color-neutral-white));
-  --osdk-color-background-button-camera-hover: rgba(
-    var(--ods-color-primary-400)
-  );
-  --osdk-color-background-button-camera-active: rgba(
-    var(--ods-color-primary-600)
-  );
+  --color-background-button-camera: rgb(var(--ods-color-neutral-white));
+  --color-background-button-camera-hover: rgb(var(--ods-color-primary-400));
+  --color-background-button-camera-active: rgb(var(--ods-color-primary-600));
```

Comment:
Should these 2 still have the `osdk` prefix?

Reply:
Yes, they should! Might have messed it up when I merged in the latest from the feature branch.